### ISSUE TITLE
Best practices on Syntax

### ISSUE THREAD

**Issue by tommygebru** [DeniseO25/prj-rev-bwfs-tea-cozy#2]
Great work on closing your html elements this will make sure the html structure follows your logic. Also, great work on using other html attributes such as id and class names, both provide clarification on how to use and access your elements. 

_Also, cool use of html alt attributes,_ it helps identify the contents of image elements within your webpage for users who can’t view the image.

**Comment by tommygebru** [comment]
**Minor typos and errors:**

- two missing _closing div tags_ from the Locations section

- Unclosed heading `H2` tag from the Footer section

- The anchor tags have a _default color of blue_, you can adjust this to inherit the webpage's text color
For example:
`nav a {color: seashell;}`

**Comment by DeniseO25** [comment]
Thank you Tommy, this really helps as I have been a having troubles with the particular issues you have stated.

**Comment by tommygebru** [comment]
Of course! no worries Denise sometimes these things require a second pair of eyes, but proper syntax and formatting is very important to get the desired output in web pages and apps. Pay attention to minor errors like spelling and typos, consider installing a “Spell-check” extension for your code editor, as well as other helpful “Linters” that check for coding errors 😉
